runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -225,7 +230,7 @@ protected void startUp() throws Exception
 		currPingFuture = hopperExecutorService.scheduleWithFixedDelay(this::pingCurrentWorld, 15, 1, TimeUnit.SECONDS);
 
 		// populate initial world list
-		updateList();
+		clientThread.invokeLater(this::updateList);
```

Comment:
calling getEnum() before the client is loaded (when the loading state is < LOGIN_SCREEN) is not valid, and can make the client cache a non-loaded enum.